extensions/proposals/WEBGL_video_texture/extension.xml

```diff
@@ -20,21 +20,21 @@
   <overview>
     <mirrors href="https://www.khronos.org/registry/gles/extensions/OES/OES_EGL_image_external.txt"
              name="OES_EGL_image_external">
-      <addendum>Defines a new texture target <code>TEXTURE_EXTERNAL_OES</code>.</addendum>
-      <addendum>Provides a mechanism for binding <code>HTMLVideoElement</code>'s EGLImage to external texture targets.</addendum>
-      <addendum>Provides time of frame, texture width and height of <code>HTMLVideoElement</code>'s EGLImage.</addendum>
+      <addendum>Defines a new texture target <code>TEXTURE_VIDEO_IMAGE</code>.</addendum>
```

Comment:
I think that we should leave open the possibility that on some platforms, the TEXTURE_VIDEO_IMAGE(_WEBGL) target is actually aliased to TEXTURE_2D. At least, this will prevent developers from assuming that it is always a different target. What do you think? If you agree, could you update the spec to allow this possibility?